docs/_documentations/project-actions.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+title: Project actions
+description: Project actions
+keywords: run, open, import, show, restart, edit, build, logs, tools, project actions, attach, build, disable, enable, validate, refresh
+duration: 1 minute
+permalink: project-actions
+type: document
+order: 30
+parent: 
+---
+
+# Project actions
+
+The following actions are available by right clicking on the project in the **Codewind Explorer** view. Most actions are only available if the project is enabled.
+
+Some actions open the default browser. If you find that the default browser cannot handle the content, change the default browser by navigating to **Window** > **Web Browser** and selecting a different browser from the list.
+
+- **Open Application**: Opens the application in the default browser. This action is only available when the application is running or debugging.
+- **Project Overview**: Opens the overview page for this project. You can use this to see information about the project, enable or disable the project, turn auto build on and off, or edit project settings.
+- **Open Container Shell**: Opens a shell into your application container. This action is only available when the container is active.
```

Comment:
Move this to below Tekton Dashboard to match the order in the menu.

Reply:
Resolved